addons/mail/models/res_config_settings.py

```diff
@@ -59,6 +59,16 @@ class ResConfigSettings(models.TransientModel):
         config_parameter='discuss.tenor_gif_limit',
         help="Fetch up to the specified number of GIF.",
     )
+    enable_message_translation = fields.Boolean(
+        'Enable Message Translation',
+        help="A 'Translate' option will be available for chatter messages.",
+        config_parameter='discuss.enable_message_translation',
+    )
+    message_translation_api_key = fields.Char(
+        'Message Translation API Key',
+        help="A valid Google API key is required to enable message translation. https://cloud.google.com/translate/docs/setup",
+        config_parameter='discuss.message_translation_api_key',
+    )
```

Comment:
Indeed, the only value of this field is to deactivate/reactivate the feature at while keeping the actual API key stored somewhere but it might not be worth. I removed it.